src/mon/MDSMonitor.cc

```diff
@@ -1551,8 +1549,9 @@ int MDSMonitor::filesystem_command(
     MDSMap::mds_info_t failed_info;
     mds_gid_t gid = gid_from_arg(fsmap, who, ss);
     if (gid == MDS_GID_NONE) {
-      ss << "MDS named '" << who << "' does not exist, is not up or you "
-	 << "lack the permission to see.";
+      ss << "Error: MDS named '" << who << "' does not exist, is not up or "
+	 << "you lack the permission to see.";
```

Comment:
Had a discussion with Greg about this. This is not a good idea because prefixing with error message with "Error" will confuse the users. Instead, we should add something indicates idempotency and yet doesn't confuse users.

Reply:
Some loosely-related previous conversation on this topic - https://github.com/ceph/ceph/pull/57333#discussion_r1594179678

Reply:
Maybe an `Alarm!` prefix instead of the `Error` prefix should help grab attention without causing a panic.

Reply:
@rishabh-d-dave If there a way forward for this then?